test/index.spec.js

```diff
@@ -25,7 +25,7 @@ describe('babel-plugin-css-modules-transform', () => {
     }
 
     function readExpected(path) {
-        return readFileSync(resolve(__dirname, path), 'utf8');
+        return readFileSync(resolve(__dirname, path), 'utf8').trim();
```

Comment:
Used this so that we remove trailing new lines from the expected files. Some (like me) have the editors configured so that the new line is automatically inserted, so figured it'd be good to just ignore that instead of having an exception for these files. If this is something you don't want, I can remove it and remove the new line in the expected file.